test/tools/libtesteth/TestSuite.cpp

```diff
@@ -55,6 +61,23 @@ void removeComments(json_spirit::mValue& _obj)
 	}
 }
 
+TestFileData readTestFile(fs::path _testFileName)
```

Comment:
I recommend passing `boost::filesystem::path` by const reference, it's what boost itself does